### ISSUE TITLE
Description tab on New User Collection page missing role context

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6797]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `Description` tab on the New User Collection page does not have the correct context role and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/dashboard/collections/new?locale=en&collection_type_id=3 and run the Site Improve browser extension

![new_collection_roles](https://github.com/samvera/hyrax/assets/32644254/a65e10ad-ab0a-4e54-8c9f-83b05da8aaef)

**Comment by aahurford** [comment]
tested and confirmed issue on Nurax-PG

**Comment by christhepianist** [comment]
Confirmed that this is no longer an issue.
